aleph-client/docker/subxt-integration-entrypoint.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+# an ugly workaround for the fact we want to ignore rustdoc warnings in generated runtime file
+echo "#[doc(hidden)]" > aleph_zero.rs
```

Comment:
This file seems to be different now on `main` branch https://github.com/Cardinal-Cryptography/aleph-node/blob/main/aleph-client/docker/subxt-integration-entrypoint.sh

Reply:
Well, this merges `main` at some particular point in time, so it might have gotten removed since.